exchanges/order/orders.go

```diff
@@ -971,12 +972,11 @@ func FilterOrdersByPairs(orders *[]Detail, pairs []currency.Pair) {
 			continue
 		}
 
```

Comment:
🥃 Note for me to sniff this func's usage.
* Could/Should be a func on a List type
* And that's if we accept it should be mutable. Idiomatically only Sort may be mutable (given slices.Sort). Otherwise we should be functional programming and returning a new slice
I'll resolve and solo this at some point. Or raise an issue